app/build.gradle.kts

```diff
@@ -47,6 +47,11 @@ androidComponents.onVariants { variant ->
             "sample_fan_native_320_id",
             "sample_gam_placement_id",
             "sample_unity_game_id",
+            "sample_vungle_config_id",
```

Comment:
These need blank definitions in the gradle.properties file in the root of the project.